packages/xsnap/lib/text-shim.js

```diff
@@ -7,14 +7,21 @@ const { fromString } = ArrayBuffer;
 const { fromArrayBuffer } = String;
 
 class TextEncoder {
-  encode(s) {
-    return new Uint8Array(fromString(s));
+  encode(text) {
```

Comment:
Changing `s` to `text` seems like a bit of a drive-by. I suppose it's a welcome change, but I prefer to see at least a mention of this sort of thing in the commit message.

Reply:
That’s fair. I’ve split the commit into the body and a follow-up: `refactor(xsnap): Expand terse argument name in text shim for local consistency`